Orange/widgets/data/owtable.py

```diff
@@ -1028,70 +1027,6 @@ def parts(array, density, col_dist):
         return Summary(n_instances, domain, X_part, Y_part, M_part)
 
 
-def format_summary(summary):
```

Comment:
Try reusing this function (or rewrite it for your needs) to get more accurate results on lengths.

Reply:
What do you think about reusing this function but within the `format_summary_details` function? That function takes the table as the input and it calculates all the lengths itself. Also, in that way we get consistent results for all widgets.

Reply:
I only meant it for the "length" part of the function. It returns `approx_len` for SqlTable, unless the length has finished calculating. I'm not sure whether we can spot the difference (probably for big sql tables), but I guess we should keep the behavior.